<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>printCodeWidget</class>
 <widget class="QWidget" name="printCodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>编码显示窗口</string>
  </property>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>编码文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrintCode">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加载编码文件并显示于窗口中</string>
   </property>
   <property name="text">
    <string>加载并显示</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>文件加载进度条</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_progress">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>等待加载文件中</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
